runtime/src/plugin.rs

```diff
@@ -2,6 +2,9 @@ use std::collections::BTreeMap;
 
 use crate::*;
 
+pub const EXTISM_ENV_MODULE: &str = "extism:env";
+pub const EXTISM_USER_MODULE: &str = "extism:user";
```

Comment:
*winces* Ack, sorry sorry. I think we might want to add an interface projection here, too: something like `extism:core/env` / `extism:core/user`. Mea culpa here – I ran into this with the [observe-api](https://github.com/dylibso/observe-sdk/pull/128/files#diff-0a8d99077aa2dacc5aa5e7d571411c1f496737d0707f978771f9d65cd0674469R3) component model support. (The `/foo` bit maps to an interface in WIT.)

Reply:
Thanks for catching this!!! I think I will go with `extism:host/env` and `extism:host/user`